Hackaton2023IOS.xcodeproj/xcuserdata/andriilytvyniuk.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -7,7 +7,7 @@
 		<key>Hackaton2023IOS.xcscheme_^#shared#^_</key>
 		<dict>
 			<key>orderHint</key>
-			<integer>0</integer>
+			<integer>3</integer>
```

Comment:
We probably don't need these changes. As an idea we can add it to git ignore

Reply:
Done